compiler/compile/OMRCompilation.hpp

```diff
@@ -575,6 +575,17 @@ class OMR_EXTENSIBLE Compilation
 
    TR::list<TR::Snippet*> *getSnippetsToBePatchedOnClassRedefinition();
 
+  /*
+   * \brief
+   *    Inserts new first block into the IL
+   *
+   * \note
+   *    Inserts an empty block before the current first block
+   *    Moves glRegDeps from the current block to the new one
```

Comment:
Should this say "Moves glRegDeps from the current first block to the new one" ?

Reply:
Done